pkg/cli/host.go

```diff
@@ -51,3 +53,16 @@ func hostnames(service client.Service) ([]string, error) {
 	}
 	return result, nil
 }
+
+func findHost(service client.Service, hostname string) (*apiv1.HostInstance, error) {
```

Comment:
nit: `name` only is ok here. (https://google.github.io/styleguide/go/decisions.html#variable-names)

Reply:
Done.